docs/events.md

```diff
@@ -392,13 +392,12 @@ Also triggers a **message** event with .type = 'privmsg'
 ~~~javascript
 {
     nick: 'prawnsalad',
-    message: 'You are now back',
+    message: 'You are now back' || '',
```

Comment:
This ` || ''` is confusing to see here. It's not needed to show every possible string here.